CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Changelog
+* Fixed a bug in the **format** command where indicatorField field.
```

Comment:
I think we can remove this because we dod not fix anything here, the fix was already mentioned in this file right?

Reply:
Fixed a bug in the **format** command where `indicatorField` wasn't part of the SDK schemas.